ironfish/src/assets/assetsVerificationApi.ts

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import axios, { AxiosError } from 'axios'
+
+type GetVerifiedAssetsResponse = {
+  data: Array<{ identifier: string }>
+}
+
+type GetVerifiedAssetsRequestHeaders = {
+  'if-modified-since'?: string
+}
+
+type GetVerifiedAssetsResponseHeaders = {
+  'last-modified'?: string
+}
+
+export class VerifiedAssets {
+  private readonly assetIds: Set<string> = new Set()
+  private lastModified?: string
+
+  isVerified(assetId: Buffer | string): boolean {
+    if (!(typeof assetId === 'string')) {
+      assetId = assetId.toString('hex')
+    }
+    return this.assetIds.has(assetId)
+  }
+}
+
+export class AssetsVerificationApi {
+  private readonly url: string
+  private readonly timeout: number
+
+  constructor(options?: { url?: string; timeout?: number }) {
+    this.url = options?.url ?? 'https://api.ironfish.network/assets?verified=true'
```

Comment:
I looked at this URL and it exists, it seems like it isn't filtering out verified assets, showing them all. Just wanted to flag, but I am sure you are aware.

Reply:
Ah, yeah. API changes are complete, but they still have to be deployed to api.ironfish.network. They're deployed to testnet.api.ironfish.network FWIW.